querybook/webapp/components/DataDocTags/CreateDataDocTag.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import { useDispatch } from 'react-redux';
+
+import { EntitySelect } from 'components/Search/EntitySelect';
+import { ITag } from 'const/tag';
+import { isTagValid } from 'lib/utils/tag';
+import { Dispatch } from 'redux/store/types';
+import { createDataDocTag } from 'redux/tag/action';
+import { DataDocTagResource } from 'resource/dataDoc';
+import { IconButton } from 'ui/Button/IconButton';
+
+import './CreateDataDocTag.scss';
+
+interface IProps {
+    datadocId: number;
+    tags: ITag[];
+}
+
+export const CreateDataDocTag: React.FunctionComponent<IProps> = ({
```

Comment:
I guess it is similar with `CreateDataTableTag`,  can we consolidate them to make it reusable, instead of duplicating the code?